df3d/core.py

```diff
@@ -412,6 +413,21 @@ def setup_camera_ordering(self, camera_ordering) -> np.ndarray:
         # self.cidread2cid, self.cid2cidread = read_camera_order(self.output_folder)
         return np.array(camera_ordering)
 
+    def get_fps(self):
+        rates = []
+        for vid in glob.glob(os.path.join(self.input_folder, "camera_?.mp4")):
+            cmd = ["ffprobe", "-v", "error", "-select_streams", "v:0",
+                   "-show_entries", "stream=avg_frame_rate", "-of",
+                   "default=noprint_wrappers=1:nokey=1", vid]
+            rates.append(subprocess.check_output(cmd, text=True))
+        if len(rates) == 0:
+            return None
+        if any(rate != rates[0] for rate in rates):
+            return None
+        # All videos returned the same rate, so we return that
+        numerator, denominator = map(int, rates[0].split('/'))
```

Comment:
What's the default output if no FPS information is saved in the video metadata? In other words, will this line fail because "/" is not detected?